### DISCUSSION TITLE
Build queed

### DISCUSSION THREAD

**Discussion by VKV-Vyvoj** [vercel/community#4252] [Help] [answered]
### Summary

I can't do the build, it still loads and then reports the voice fetch error vercel

### Steps to Reproduce

I can't do the build, it still loads and then reports the voice fetch error vercel

**Comment by ofhouse** [comment, marked as answer]
We had an issue with builds for Hobby & Pro customers and are actively working on mitigation.
Please see more information on our status page: https://www.vercel-status.com/incidents/87m1ypq44hf0
